From 6416a8f5a82f85c887fd290f58e15e803a8cd743 Mon Sep 17 00:00:00 2001
From: kbuild test robot <fengguang.wu@intel.com>
Date: Mon, 4 Aug 2014 13:49:40 +1000
Git-commit: 6eee8972cc1799d3dde3f1e699abf286ed3adbaf
Patch-mainline: v3.17
References: bsc#1035531
Subject: [PATCH 2/2] xfs: fix coccinelle warnings

Removes unneeded semicolon, introduced by commit a70a4fa5 ("xfs: fix
a couple error sequence jumps in xfs_mountfs"):

fs/xfs/xfs_mount.c:858:24-25: Unneeded semicolon

Generated by: scripts/coccinelle/misc/semicolon.cocci

Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Reviewed-by: Brian Foster <bfoster@redhat.com>
Signed-off-by: Dave Chinner <david@fromorbit.com>
Signed-off-by: Ales Novak <alnovak@suse.cz>
---
 fs/xfs/xfs_mount.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/xfs/xfs_mount.c b/fs/xfs/xfs_mount.c
index 59ea9481cc9d..a9991b41d8d5 100644
--- a/fs/xfs/xfs_mount.c
+++ b/fs/xfs/xfs_mount.c
@@ -855,7 +855,7 @@ xfs_mountfs(
 	     !mp->m_sb.sb_inprogress) {
 		error = xfs_initialize_perag_data(mp, sbp->sb_agcount);
 		if (error)
-			goto out_log_dealloc;;
+			goto out_log_dealloc;
 	}
 
 	/*
-- 
2.11.0.258.ge05806d

